Tests/HashTests/HashTests.swift

```diff
@@ -0,0 +1,95 @@
+import XCTest
+import Core
+@testable import Hash
+
+class HashTests: XCTestCase {
+    static var allTests = [
+        ("testSHA1", testSHA1),
+        ("testSHA224", testSHA224),
+        ("testSHA384", testSHA384),
+    ]
+
+    func testSHA1() throws {
```

Comment:
The websocket test doesn't seem to be in here anymore, I think that's a great one to have since it broke the last implementation.

```
"dGhlIHNhbXBsZSBub25jZQ==258EAFA5-E914-47DA-95CA-
         C5AB0DC85B11"
=>
[0xb3 0x7a 0x4f 0x2c 0xc0 0x62 0x4f 0x16 0x90 0xf6
         0x46 0x06 0xcf 0x38 0x59 0x45 0xb2 0xbe 0xc4 0xea]
=> Base64
s3pPLMBiTxaQ9kYGzzhZRbK+xOo=
```